modules/@apostrophecms/log/index.js

```diff
@@ -313,10 +313,11 @@ module.exports = {
 
         self.processRequestData(req, data);
         // Don't override system properties.
-        delete obj.module;
-        delete obj.type;
-        delete obj.severity;
-        Object.assign(data, obj);
+        Object.assign(data, obj, {
```

Comment:
I think just doing three property assignments in a row would be faster than creating an object and passing it to `Object.assign`.